datasets/timit_asr/README.md

```diff
@@ -0,0 +1,232 @@
+
+### Supported Tasks and Leaderboards
+
+- `automatic-speech-recognition`, `speaker-identification`: The dataset can be used to train a model for Automatic Speech Recognition (ASR). The model is presented with an audio file and asked to transcribe the audio file to written text. The most common evaluation metric is the word error rate (WER). The task has an active leaderboard which can be found at https://paperswithcode.com/sota/speech-recognition-on-timit and ranks models based on their WER.
+
+### Languages
+
+The audio is in English. 
+The TIMIT corpus transcriptions have been hand verified. Test and training subsets, balanced for phonetic and dialectal coverage, are specified. Tabular computer-searchable information is included as well as written documentation.
+
+## Dataset Structure
+
+### Data Instances
+
+A typical data point comprises the path to the audio file, usually called `file` and its transcription, called `text`. Some additional information about the speaker and the passage which contains the transcription is provided.
+
+```
+{
+'file': wav_path,
+'text': 'Would such an act of refusal be useful?',
+'phonetic_detail': phonemes,
+'word_detail': words,
+'dialect_region': 'DR4',
+'sentence_type': 'SI',
+'speaker_id': 'MMDM0',
+'id': '681'
+```
+
+
+### Data Fields
+
+- file: A path to the downloaded audio file in .wav format.
+
+- text: The transcription of the audio file.
+
+- phonetic_detail: The phonemes that make up the sentence. The PHONCODE.DOC contains a table of all the phonemic and phonetic symbols used in TIMIT lexicon.
+
+- word_detail: Word level split of the transcript.
+
+- dialect_region: The dialect code of the recording.
+
+- sentence_type: The type of the sentence - 'SA':'Dialect', 'SX':'Compact' or 'SI':'Diverse'.
+
+- id: Unique id of the data sample. Contains the <SENTENCE_TYPE><SENTENCE_NUMBER>.  
+
+- speaker_id: Unique id of the speaker. The same speaker id can be found for multiple data samples.
+
+
+### Data Splits
+
+The speech material has been subdivided into portions for training and
+testing.  The criteria for the subdivision is described in the file
+"testset.doc".  THIS SUBDIVISION HAS NO RELATION TO THE DATA DISTRIBUTED ON
+THE PROTOTYPE VERSION OF THE CDROM.
```

Comment:
Can you add a link to the mentioned testset.doc ?

Also I find the text in capital a bit out of context. Could you add more context so that it doesn't sound too confusing ? Or we can simply remove the sentence in capital.